components/Summary/SummaryList.jsx

```diff
@@ -1,7 +1,19 @@
 import PropTypes from 'prop-types';
 
-const SummaryList = ({ list }) => (
+const SummaryList = ({ list, creation }) => (
   <dl className="govuk-summary-list">
+    {creation && (
```

Comment:
This is creating the "created by" in every single section.

Maybe we need input from design on how to solve this problem